config/database.yml

```diff
@@ -0,0 +1,25 @@
+# SQLite. Versions 3.8.0 and up are supported.
+#   gem install sqlite3
+#
+#   Ensure the SQLite 3 gem is defined in your Gemfile
+#   gem "sqlite3"
+#
+default: &default
+  adapter: postgresql
+  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  timeout: 5000
+
+development:
+  <<: *default
+  database: reloj_db_development
+
+# Warning: The database defined as "test" will be erased and
+# re-generated from your development database when you run "rake".
+# Do not set this db to the same as development or production.
+test:
+  <<: *default
+  database: reloj_db_test
+
+production:
+  <<: *default
+  database: reloj_db_production
```

Comment:
Do not upload to GitHub the database.yml, even if there's not a really critical databasa info, because in a future or in other projects maybe you put real credentials, someone can steale the info. Instead, use a file name database.yml. Add this to the .gitignore

Reply:
I agree with @germanruzca